pkgs/development/interpreters/php/8.1.nix

```diff
@@ -5,12 +5,28 @@ let
     version = "8.1.29";
     hash = "sha256-h6YDEyY/L1M/GA5xknLKXkfNmITU7DyTcgGY6v+uCCc=";
     extraPatches = [
-      # Fix build with libxml 2.12+.
+      # Fix build with libxml2 2.12+.
       # Patch from https://github.com/php/php-src/commit/0a39890c967aa57225bb6bdf4821aff7a3a3c082
       (fetchpatch {
         url = "https://github.com/php/php-src/commit/0a39890c967aa57225bb6bdf4821aff7a3a3c082.patch";
         hash = "sha256-HvpTL7aXO9gr4glFdhqUWQPrG8TYTlvbNINq33M3zS0=";
       })
+      # Fix tests with libxml2 2.12
+      (fetchpatch {
+        url = "https://github.com/php/php-src/commit/061058a9b1bbd90d27d97d79aebcf2b5029767b0.patch";
+        hash = "sha256-0hOlAG+pOYp/gUU0MUMZvzWpgr0ncJi5GB8IeNxxyEU=";
+        excludes = [
+          "NEWS"
+        ];
+      })
+      # Backport of PHP_LIBXML_IGNORE_DEPRECATIONS_START and PHP_LIBXML_IGNORE_DEPRECATIONS_END
```

Comment:
What commit does this come from?

Reply:
@jtojnar I'll PR actual commit patch instead of file. I'm just waiting for build to finish.